samples/dapr/ServiceB/DaprServiceB.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Dapr.AspNetCore" />
```

Comment:
It might not be necessary in the case of Dapr, but I am wondering whether we should be creating a `Aspire.Dapr` package for the service side like we do with other Aspire components. For example should we be automatically wiring up some telemetry and healthchecks for dapr interactions within the service code for consistency.

Adding @eerhardt for his thoughts on this.

Reply:
I think there are definitely opportunities to create Aspire components specific to Dapr, that wrap the Dapr .NET SDK is ways that make sense to tie into Aspire features (service discovery, perhaps?).  I haven't had an opportunity to  look closely at Aspire components and what it really means to create one.  For example, Dapr already ties into Open Telemetry at the sidecar level, which is ultimately doing everything on behalf of the application, so I don't know if there would be anything specific to wrap in the SDK.

Reply:
OK ... we don't need to solve it in this PR I don't think given Dapr already integrates very well with DI.

Reply:
We discussed in the past having "platform components"

https://github.com/dotnet/aspire/issues/242
https://github.com/dotnet/aspire/issues/243

It would seem this would fit in with that.